fe2-android/app/src/main/java/com/github/dedis/popstellar/model/objects/Wallet.java

```diff
@@ -37,7 +38,7 @@
  * This class represent a wallet that will enable users to store their PoP tokens with reasonable,
  * realistic security and usability.
  */
-public class Wallet {
+public class Wallet implements Serializable {
```

Comment:
Why does the wallet need to be serializable ?

Reply:
I might have added it by error, I'm gonna check whether it is needed or not